runtime/near-vm-runner/src/runner.rs

```diff
@@ -47,6 +49,48 @@ pub fn run<'a>(
     )
 }
 
+/// Runs a batch of contracts.
+///
+/// Re-using the same `WasmMachine` is potentially more efficient than calling
+/// `run` repeatedly.
+pub struct WasmMachine {
+    _private: (),
+}
+
+impl WasmMachine {
```

Comment:
`WasmMachine` is a bit misleading, maybe once we shall rename it. IMO, technically it's more like `ContractRunner` or `ContractExecutor`, isn't it?